### ISSUE TITLE
Versatile rolls don't work

### ISSUE THREAD

**Issue by DarKDinDoN** [RedReign/FoundryVTT-BetterRolls5e#13]
Hi, thanks for the last version!
I was checking my character sheet to make sure everything works fine. I noticed hat versatile rolls don't work.
When I click the checkbox in the BetterRolls5e Tab, then roll from the sheet icon, the damage roll use the "normal" value, not the versatile value.

**Comment by RedReign** [comment]
Hey! Thanks for the report. This fix will be implemented in the next update.
For now you can change the brackets in lines 649 and 657.
Each line should be this:
`let isVersatile = (i == 0) && flagIsTrue("quickVersatile");`

**Comment by DarKDinDoN** [comment]
Thx for the hotfix!

**Comment by MrPrimate** [comment]
Hi, I also noticed this error. While your hotfix fixes the alt roll, if I click the Versatile Damage button on say a spell I get the following console error:
```
betterrolls5e.js:334 Uncaught ReferenceError: forceCrit is not defined
    at setDamage (betterrolls5e.js:334)
    at HTMLButtonElement.<anonymous> (betterrolls5e.js:354)
    at HTMLButtonElement.dispatch (jquery.min.js:2)
    at HTMLButtonElement.v.handle (jquery.min.js:2)
```

**Comment by RedReign** [comment]
Both issues have been fixed in v1.0.1. Thanks for the reports @MrPrimate @DarKDinDoN !
